README.rst

```diff
@@ -84,7 +84,7 @@ A Debian/Ubuntu package is also available. Install as follows:
 
 .. code-block:: console
 
-    $ echo "deb https://apt-mo.trafficmanager.net/repos/azure-cli/ wheezy main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
+    $ echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/azure-cli/ wheezy main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
```

Comment:
I now provide separate steps for both 32 and 64 bit.
@tjprescott please review again.
Thanks!